R/pkg/R/mllib.R

```diff
@@ -269,9 +349,29 @@ setMethod("summary", signature(object = "NaiveBayesModel"),
             return(list(apriori = apriori, tables = tables))
           })
 
+#' Makes predictions from a model produced by spark.naiveBayes(),
+#' similarly to R package e1071's predict.
+#'
+#' @param object A fitted naive Bayes model
+#' @param newData SparkDataFrame for testing
+#' @return SparkDataFrame containing predicted labels in a column named "prediction"
+#' @rdname predict
+#' @name predict
+#' @export
+#' @examples
+#' \dontrun{
+#' model <- spark.naiveBayes(trainingData, y ~ x)
+#' predicted <- predict(model, testData)
+#' showDF(predicted)
+#'}
+setMethod("predict", signature(object = "NaiveBayesModel"),
+          function(object, newData) {
+            return(dataFrame(callJMethod(object@jobj, "transform", newData@sdf)))
+          })
+
 #' Fit a k-means model
 #'
-#' Fit a k-means model, similarly to R's kmeans().
+#' Fits a k-means model, similarly to R's kmeans().
```

Comment:
seems to be changing a few times here - should this be `Fits a ...` or `Fit a ...`?